src/modules/household/components/ManageHousehold.tsx

```diff
@@ -196,12 +196,7 @@ const ManageHousehold = ({
 
           {searchInput && (
             <SsnDobShowContextProvider>
-              <Box
-                sx={{
-                  border: (theme) => `1px solid ${theme.palette.borders.light}`,
-                  borderRadius: 1,
-                }}
-              >
+              <Paper>
```

Comment:
@gigxz I'm proposing to just replace with `Paper` here to avoid custom styling, but was there a purposeful reason for this? (e.g. maybe a Paper within another Paper we consider an antipattern?)

Reply:
looks good, thanks! it was floating border-less before, which I think looked especially off with the action menu.